server/message_queue_service.py

```diff
@@ -48,9 +52,27 @@ async def initialize(self) -> None:
             await self._connect()
         except ConnectionAttemptFailed:
             return
-        self._is_ready = True
 
         await self._declare_exchange(config.MQ_EXCHANGE_NAME, ExchangeType.TOPIC)
+        await self._connect_listening_queues()
+        self._is_ready = True
+
+    async def _connect_listening_queues(self):
+        for listening_info in self._listening_info:
+            await self._declare_listening_queue(listening_info)
+
+    async def listen_to_message(self, queue_name: str, routing_key: str,
+                                callback: Callable[[AbstractIncomingMessage], Any]):
```

Comment:
We format like this to avoid long lines:
```suggestion
    async def listen_to_message(
        self, 
        queue_name: str, 
        routing_key: str,
        callback: Callable[[AbstractIncomingMessage], Any]
    ):
```

Reply:
Also it makes git diffs smaller when you add/remove parameters which makes merge conflicts less common and easier to resolve.